content/References/IPv6.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Understanding IPv6"
+description: "Provides information general information on IPv6 and implementation guidance."
+weight: 20
+tags:
+- network
+- interfaces
+- ipv6
+---
+
+## What is IPv6?
+IPv6, referred to as the Internet protocol next generation (IPng), is a 128-bit alphanumeric address organized into eight groups of 16 bits with each group expressed as four-digit hexadecimal numbers separated by colons.
+An example of an IPv6 address is *fe80:cd00:0000:0cde:1257:0000:211e:729c*.
+
+IPv6 addresses are split into two parts: a network (routing prefix) and a node component(interface ID).
+
+The network component is the first 64 bits of the address and is used for routing.
```

Comment:
This is too specific.  The network component is commonly 64 bits, but doesn't have to be.

Reply:
reworded to be more generalized.